examples/github_search/flutter_github_search/lib/main.dart

```diff
@@ -27,9 +27,9 @@ class App extends StatelessWidget {
     return MaterialApp(
       title: 'Github Search',
       home: Scaffold(
-        appBar: AppBar(title: Text('Github Search')),
-        body: BlocProvider(
-          create: (context) =>
+        appBar: AppBar(title: const Text('Github Search')),
+        body: BlocProvider<GithubSearchBloc>(
```

Comment:
```suggestion
        body: BlocProvider(
```

this isn't necessary as it's inferred.